### DISCUSSION TITLE
TDesignBlazor  v0.1 吐槽大会

### DISCUSSION THREAD

**Discussion by teacher-zhou** [AchievedOwner/TDesignBlazor#101] [公告]
## 📢 TDesignBlazor 第一个版本已经发布
如果需要，请阅读完整的[发行说明](https://github.com/AchievedOwner/TDesignBlazor/releases/tag/0.1)

组件需要逐步的完善，但我们更想要得到用户的反馈，以帮助我们及时改进：

* 如果你遇到问题，请移步 [Github Issue](https://github.com/AchievedOwner/TDesignBlazor/issues) 提出
* 如果您愿意加入到贡献者，请先阅读[贡献指南](https://github.com/AchievedOwner/TDesignBlazor/blob/master/Contributing.md)

💯  0.1 正式版本发布

**利他和开放是我们的价值观**

**Comment by XiaoHeitu** [comment]
TDesign都还在测试阶段，你就出 blazor版了。。。太快了吧。

**Reply by teacher-zhou** [reply]
> TDesign都还在测试阶段，你就出 blazor版了。。。太快了吧。

其实问题并不大，Blazor 不过是把行为进行了包装，样式还是那些
